package.json

```diff
@@ -2,6 +2,7 @@
   "version": "4.24.0",
   "private": true,
   "license": "MIT",
+  "homepage": "https://github.com/algolia/algoliasearch-client-javascript#readme",
```

Comment:
shouldn't this be in the package.json of `algoliasearch` instead of the root one? Also I wonder if it shouldn't be the documentation link instead, as the repo is already linked too

Reply:
yes plus it's targeting v4, we don't release anything on this version except security/bug fixes

@Adnan-Bacic would you mind porting the PRs to https://github.com/algolia/api-clients-automation instead?

Reply:
i dont quite understand why, but okay

Reply:
This whole repository is generated, and the source is from https://github.com/algolia/api-clients-automation, once your other PR is merged it will be reflected here